css-conditional-5/Overview.bs

```diff
@@ -1067,6 +1087,159 @@ Style Container Features</h3>
+		<dt><dfn>top</dfn>
+		<dd>
+			The ''position/sticky'' container is shifted to stay inside the top edge.
+		<dt><dfn>right</dfn>
+		<dd>
+			The ''position/sticky'' container is shifted to stay inside the right edge.
+		<dt><dfn>bottom</dfn>
+		<dd>
+			The ''position/sticky'' container is shifted to stay inside the bottom edge.
+		<dt><dfn>left</dfn>
+		<dd>
+			The ''position/sticky'' container is shifted to stay inside the left edge.
+		<dt><dfn>block-start</dfn>
+		<dd>
+			The ''position/sticky'' container is shifted to stay inside the block-start edge.
+		<dt><dfn>inline-start</dfn>
+		<dd>
+			The ''position/sticky'' container is shifted to stay inside the inline-start edge.
+		<dt><dfn>block-end</dfn>
+		<dd>
+			The ''position/sticky'' container is shifted to stay inside the block-end edge.
+		<dt><dfn>inline-end</dfn>
+		<dd>
+			The ''position/sticky'' container is shifted to stay inside the inline-end edge.
+	</dl>
+
+<h4 id="snapped">
+Scroll snapping: the '@container/snapped' feature</h4>
+
+	<pre class="descdef mq">
+		Name: snapped
+		For: @container
+		Value: x | y | block | inline
+		Type: discrete
+	</pre>
+
+	The '@container/snapped' [=container feature=] queries whether a [=snap target=]
+	is snapped to its [=snap container=] in the given axis. It matches in the boolean
+	context if it is snapped in at least one of the directions.
+
+	<dl dfn-type=value dfn-for="@container/snapped">
+		<dt><dfn>x</dfn>
+		<dd>
+			'@container/snapped' [=container feature=] matches ''x''
+			if the container is a horizontal [=snap target=] for its [=scroll container=]
+		<dt><dfn>y</dfn>
+		<dd>
+			'@container/snapped' [=container feature=] matches ''y''
+			if the container is a vertical [=snap target=] for its [=scroll container=]
+		<dt><dfn>block</dfn>
+		<dd>
+			'@container/snapped' [=container feature=] matches ''block''
+			if the container is a [=snap target=] for its [=scroll container=]
+			in the block direction of the [=snap container=].
+		<dt><dfn>inline</dfn>
+		<dd>
+			'@container/snapped' [=container feature=] matches ''inline''
+			if the container is a [=snap target=] for its [=scroll container=]
+			in the inline direction of the [=snap container=].
+	</dl>
+
+<h4 id="overflowing">
+Sticky positioning: the '@container/overflowing' feature</h4>
+
+	<pre class="descdef mq">
+		Name: overflowing
+		For: @container
+		Value: top | right | bottom | left | block-start | inline-start | block-end | inline-end
+		Type: discrete
+	</pre>
+
+	The '@container/overflowing' [=container feature=] queries whether a
+	[=scroll container=] has [=scrollable overflow=] in the given direction for the
+	current scroll progress. The logical values map to physical based on the direction
+	and writing-mode of the [=query container=].
+
+	In the boolean context, the query matches if there is scrollable overflow in any
+	direction.
+
+	<dl dfn-type=value dfn-for="@container/overflowing">
+		<dt><dfn>top</dfn>
+		<dd>
```

Comment:
I think it should be clear if these match when there is overflow even if the container is not scrollable in that direction.

I also wonder if there's demand for both variants, that is, an `(overflowing:left)` that is true only for RTL or vertical-rl pages where you can actually scroll to the left, and a differently named `(overflowing: left)` that is true if there is overflow to the left whether or not it can be scrolled to.

(I don't think it's worth distinguishing based on `overflow: hidden`... particularly since that's *programatically* scrollable.  But if things do depend on "is it scrollable in that direction" it should be clear whether or not `hidden` makes a difference.)

Reply:
> I think it should be clear if these match when there is overflow even if the container is not scrollable in that direction.
> 
> I also wonder if there's demand for both variants, that is, an `(overflowing:left)` that is true only for RTL or vertical-rl pages where you can actually scroll to the left, and a differently named `(overflowing: left)` that is true if there is overflow to the left whether or not it can be scrolled to.

The intention is that matching is based on the current scroll position, so `left` and `right` will both match when a horizontal scrollbar is not a 0 nor at the max position. If there is content overflowing to the left of an ltr container, the content is not reachable through user scrolling, and `(overflowing:left)` will not match.

> (I don't think it's worth distinguishing based on `overflow: hidden`... particularly since that's _programatically_ scrollable. But if things do depend on "is it scrollable in that direction" it should be clear whether or not `hidden` makes a difference.)

I used the term [_scrollable overflow_](https://drafts.csswg.org/css-overflow-3/#scrollable-overflow) to exclude `overflow:hidden` and overflow that's not reachable through user scrolling. I've made it more explicit in the text now.